aiohttp_pydantic/__init__.py

```diff
@@ -1,5 +1,5 @@
 from .view import PydanticView
 
-__version__ = "1.7.2"
+__version__ = "1.7.3"
```

Comment:
Maybe this should go to `1.8.0` if we add new functionality?

Reply:
Yes you can set `__version__` to 1.8.0